pkgs/top-level/all-packages.nix

```diff
@@ -26444,4 +26444,6 @@ in
 
   jitsi-meet-electron = callPackage ../applications/networking/instant-messengers/jitsi-meet-electron { };
 
+  mailspring = callPackage ../applications/networking/mailreaders/mailspring {};
```

Comment:
This should be inserted alphabetically-ish. I would probably place it near thunderbird, there might be other mail clients in there? ``all-packages`` is loosely sorted by category then name.

Reply:
I placed it near thunderbird and mutt.